cmd/prow-job-dispatcher/main.go

```diff
@@ -7,22 +7,33 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
+	"sync"
 	"time"
 
 	prometheusapi "github.com/prometheus/client_golang/api/prometheus/v1"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/semaphore"
+	k8score "k8s.io/api/core/v1"
```

Comment:
nit: This is usually imported as `corev1`

Reply:
Done.